pulp_smash/__main__.py

```diff
@@ -16,33 +16,45 @@
     '''\
     {
         "pulp": {
-            "base_url": "https://pulp.example.com",
             "auth": ["username", "password"],
-            "verify": true,
-            "version": "2.7.5",
-            "cli_transport": "local"
+            "version": "2.12.2"
         },
-        "pulp agent": {
-            "base_url": "https://pulp-agent.example.com"
+        "systems": {
+            "pulp.example.com": {
+                "amqp broker": {"service": "qpidd"},
+                "api": {
+                    "scheme": "https",
+                    "verify": true
```

Comment:
auth is a global option, because that it is inside the pulp section not for each system